lib/Dialect/Verif/Transforms/VerifyClockedAssertLike.cpp

```diff
@@ -86,12 +86,7 @@ struct VerifyClockedAssertLikePass
       if (!defOp || handledOps.contains(defOp))
         continue;
 
-      // This is triggered if our operand is already in the worklist and
-      // wasn't handled
-      if (!worklist.insert({defOp, defOp->operand_begin()}).second) {
```

Comment:
A bit of a subtlety of this verfier, the one way I usually get rid of dependency cycles is by pre-visiting the this that would support that, however here that isn't really necessary, as we just want to make sure that none of the children ops are clocks, so I removed that to avoid having errors thrown when the IR was valid.

Reply:
basically dependency cycles aren't an issue here afaik